rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-criteria.js

```diff
@@ -136,7 +136,7 @@ export class SakaiRubricCriteria extends SakaiElement {
     criterion.ratings = sortedIds.map(id => criterion.ratings.find(r => r.id == id));
 
     var baseUrl = `${window.location.protocol}//${window.location.host}/rubrics-service/rest/ratings/`;
-    const urlList = sortedIds.reverse().reduce((a, id) => { return `${baseUrl}${id}\n${a}` }, '');
+    const urlList = sortedIds.reverse().reduce((a, id) => { return `${baseUrl}org.sakaiproject.rubrics:rubrics-service:war:20-SNAPSHOT\n${a}` }, '');
```

Comment:
is this change purposeful?

Reply:
I'm glad you spotted that. That is some weird paste error. Nice one!